### DISCUSSION TITLE
Unable to fetch data...

### DISCUSSION THREAD

**Discussion by stuu3k** [Lissy93/dashy#440] [Help and Support] [answered]
Hello,

Loving Dashy thanks! 👍 

I can't seem to get any of the API's to connect though I am running from a Synology NAS Docker Container which is constnatly connected toa  VPN, but the docker container has a local DNS server configured and can resolve local DNS names ok, on top of that Ping Checks to said hosts work, hower the API's always result in Unable ot Fetch data.

I have tried all the Pi-Hole plugins on 3 servers with DNS Names and IP's with and without /admin.
I then tried Glances Plugin for a Network Traffic amongst others.
Finally I just tried NetData Plugin with my OPNSense boxes and again no luck.

It says check logs but and IDK if I missed it in everything I have read on this but I can't see how to access the logs?

Maybe then I can figure out what the problem is unless there is anything else I shoudl be checking first?

Thanks

Stuart.

**Comment by lissy93** [comment]
Heya,
Hmm, this is weird. Could you share the logs with me so I can figure out what's going on? You can find them in the browser console, [here are the instructions](https://github.com/Lissy93/dashy/blob/master/docs/troubleshooting.md#how-to-open-browser-console).

I'm guessing that it will be a CORS error, which a couple other users have faced also. If so, the solution should be pretty simple, just set the correct headers and / or set the `useProxy` option. Did you see the [Widget Troubleshooting Guide](https://github.com/Lissy93/dashy/blob/master/docs/widgets.md#troubleshooting-widget-errors) I put in the docs?

**Comment by stuu3k** [comment, marked as answer]
Ah I was using the 'useProxy' switch incorrectly, some reason I thought it was a glabal setting and I had it on the top of the page only, I have now added it to the widgets and have some working, time to figure out the others now, thanks! 👍
